src/AppBundle/Controller/BlogController.php

```diff
@@ -34,16 +34,24 @@
 class BlogController extends Controller
 {
     /**
-     * @Route("/", defaults={"page": "1"}, name="blog_index")
-     * @Route("/page/{page}", requirements={"page": "[1-9]\d*"}, name="blog_index_paginated")
+     * @Route("/", defaults={"page": "1", "_format"="html"}, name="blog_index")
+     * @Route("/rss.xml", defaults={"page": "1", "_format"="xml"}, name="blog_rss")
+     * @Route("/page/{page}", defaults={"_format"="html"}, requirements={"page": "[1-9]\d*"}, name="blog_index_paginated")
      * @Method("GET")
      * @Cache(smaxage="10")
+     *
+     * NOTE: For standard formats, Symfony will also automatically choose the best
+     * Content-Type header for the response.
+     * See http://symfony.com/doc/current/quick_tour/the_controller.html#using-formats
      */
-    public function indexAction($page)
+    public function indexAction($page, $_format)
```

Comment:
We lost then the auto `Content-Type` header response in `blog_rss` route, remember we need the special variable `_format` into `defaults` option. ;)